src/v2/Apps/Conversation/routes.tsx

```diff
@@ -26,7 +26,7 @@ export const conversationRoutes: RouteConfig[] = [
   {
     path: "/user/conversations/:conversationID",
     displayFullPage: true,
-    getComponent: () => loadable(() => import("./Routes/Conversation")),
+    Component: loadable(() => import("./Routes/Conversation")),
```

Comment:
Why change to `Component`?

Reply:
I leave it to @MounirDhahri to explain the details but as far as I understood this change fixed the bug of `showDetails` being reset when navigating between conversations. (cc: @zephraph since you have set this up)

Reply:
When I checked the profiler I found out that `showDetails` is being reset because the conversation is getting mounted again whenever the user navigates to a new conversation, I first tried to wrap it with a memo to avoid that but found out that `getComponent` was the issue since React wasn't able to tell if that it's the same component or not to avoid rendering it.

Reply:
🧠